game/board.py

```diff
@@ -162,6 +163,12 @@ def _can_find_gold(self, row, col, gold_position, visited):
                 return True
         return False
 
+    def sort_possibles_position(self, positions, destination_row, destination_col) -> list:
+        def sorted_key(position) -> int:
+            row, col = position
+            return ((destination_row - (row + 0.1)) ** 2 + (destination_col - col) ** 2) ** (0.5)
```

Comment:
Add a comment here explaining your math
```suggestion
            # Pythagorean distance √((d_row - row)²+(d_col -col)²)
            return ((destination_row - (row + 0.1)) ** 2 + (destination_col - col) ** 2) ** (0.5)
```
Also, why add `0.1`?

Reply:
I forgot to explain that, the 0.1
The `0.1` ensure us an advantage at the time to choose col direction over row direction in the searching.
I'll gonna add both things in the comments